### ISSUE TITLE
GH_TOKEN is leaked

### ISSUE THREAD

**Issue by azu** [jirutka/rake-jekyll#8]
@jirutka Hi, I found security issue.

https://github.com/jirutka/rake-jekyll/blob/908a0962afadc8e20ad6f5d8372545c595b99b84/lib/rake-jekyll/git_deploy_task.rb#L55

``` ruby
def push(remote_url, branch)
    sh "git push -q #{remote_url} #{branch}:#{branch}"
end
```

This code show `GH_TOKEN` in Travis CI console...

![2015-06-08_23-57-52](https://cloud.githubusercontent.com/assets/19714/8037728/86f4dff2-0e3a-11e5-93af-35d8f4aced85.jpg)

**Comment by razor-x** [comment]
This is pretty serious. I recommend switching to the deploy key method (see #6), but for a fix, wrapping this like

``` ruby
verbose false do
  sh(...)
end
```

should suppress the output.

**Comment by azu** [comment]
https://github.com/jirutka/rake-jekyll/blob/908a0962afadc8e20ad6f5d8372545c595b99b84/lib/rake-jekyll/git_deploy_task.rb#L19
I notice that `git clone` has the same problem.( and `-q` option is necessary?)

**Comment by jirutka** [comment]
Fixed in v1.0.4.

Sorry for late response, I’ve been quite busy. Thanks @azu for the report and @razor-x for the tip.

**Comment by azu** [comment]
@jirutka Thanks! :+1:
